### DISCUSSION TITLE
Weird on/offline pattern and log messages, I need help!

### DISCUSSION THREAD

**Discussion by markusdd** [teslamate-org/teslamate#3083] [Q&A]
Hi guys. I observed a relatively high phantom drain on my 85D Model S and the state graph looks like this on a day when the vehicle is not being moved. That does not look right to me. It is a pre-FL car, but has the MCU2 upgrade (Atom based), so there is no energy save setting like on the legacy ones.

Therefore, I have attached the whole log of the 21st of January. Any insights? What am I doing wrong?

Infos about setup: Teslamate latest/greatest, ruinning in a Proxmox LXC container based on Debian Bullseye.

![grafik](https://user-images.githubusercontent.com/25175069/213911208-1d7f76cc-ca36-4026-b6e7-4ef2582d1bba.png)

What concerns me are all the error messages and also teh frequent occurence of this 'stale stream data'.
Apart from that everything works fine. The vehicle and also the data logging of tesla mate.
But something keeps the car awake, which causes a concerning amount of phantom drain.

[log21Jan23.txt](https://github.com/adriankumpf/teslamate/files/10474176/log21Jan23.txt)

**Comment by leandreeberhard** [comment]
I've been having the same issue as well for around a month and a half after upgrading my 2017 Model S 100D to MCU2 and HW3. The MCU upgrade + Teslamate errors seems to be a common thread based on what I've seen. Here's a screenshot from my Teslamate, which looks very similar to yours. The error in the log also seems to be identical.

Were you ever able to find a solution to this?

<img width="1535" alt="Screenshot 2023-06-19 at 14 44 47"
 src="https://github.com/adriankumpf/teslamate/assets/71736220/f660c894-024a-4203-b962-98f8ed1dbbd1">

[log2023-06-19.txt](https://github.com/adriankumpf/teslamate/files/11791319/log2023-06-19.txt)

**Comment by markusdd** [comment]
see here https://github.com/adriankumpf/teslamate/issues/3084
